build-tools/installers/create-installers.targets

```diff
@@ -20,6 +24,60 @@
   </PropertyGroup>
+  <ItemGroup>
+    <_MsxDocAssembly Include="Mono.Android">
+      <SourceFolder>$(_AndroidApiDocsPath)docs\%(Identity)\en\</SourceFolder>
+    </_MsxDocAssembly>
+    <_MonoDocAssembly Include="@(_MsxDocAssembly);OpenTK-1.0;Xamarin.Android.NUnitLite">
+      <SourceFolder>$(_AndroidApiDocsPath)docs\%(Identity)\en\</SourceFolder>
+    </_MonoDocAssembly>
+  </ItemGroup>
+  <Target Name="_FindFrameworkDirs">
+    <ItemGroup>
+      <_FrameworkDirs Include="@(AndroidApiInfo->'$(FrameworkSrcDir)%(Identity)\')" />
+      <_FrameworkDirsThatExist Condition="Exists('%(Identity)')" Include="@(_FrameworkDirs)" />
+      <_EarlierFrameworkDir Include="@(_FrameworkDirsThatExist)" Exclude="$(_LatestStableFrameworkDir)" />
+    </ItemGroup>
+  </Target>
+  <Target Name="_FindDocSourceFiles">
+    <ItemGroup>
+      <_MsxDocSourceFile Include="%(_MsxDocAssembly.SourceFolder)**" />
+      <_MonoDocSourceFile Include="%(_MonoDocAssembly.SourceFolder)**" />
+    </ItemGroup>
+  </Target>
+  <Target Name="_GenerateMsxDocXmls"
+      DependsOnTargets="_FindFrameworkDirs;_FindDocSourceFiles"
+      Inputs="@(_MsxDocSourceFile)"
+      Outputs="@(_MsxDocAssembly->'$(_LatestStableFrameworkDir)%(Identity).xml')">
+    <Exec Command="mdoc --debug export-msxdoc -o &quot;$(_LatestStableFrameworkDir)%(_MsxDocAssembly.Identity).xml&quot; &quot;%(_MsxDocAssembly.SourceFolder)&quot;" />
+  </Target>
+  <Target Name="_GenerateMsxDocXmlRedirects"
+      DependsOnTargets="_FindFrameworkDirs"
+      Inputs="$(XamarinAndroidSourcePath)build-tools\scripts\redirect-Mono.Android.xml.in"
+      Outputs="@(_EarlierFrameworkDir->'%(Identity)Mono.Android.xml')">
+    <ReplaceFileContents
+        SourceFile="$(XamarinAndroidSourcePath)build-tools\scripts\redirect-Mono.Android.xml.in"
+        DestinationFile="%(_EarlierFrameworkDir.Identity)Mono.Android.xml"
+        Replacements="@LAST_FRAMEWORK_VERSION@=$(AndroidLatestStableFrameworkVersion)" />
+  </Target>
+  <Target Name="_GenerateMonoAndroidLibArchive"
+      DependsOnTargets="_FindDocSourceFiles"
+      Inputs="@(_MonoDocSourceFile)"
+      Outputs="$(_MonoDocOutputPath)MonoAndroid-lib.zip;$(_MonoDocOutputPath)MonoAndroid-lib.tree">
+    <MakeDir Directories="$(_MonoDocOutputPath)" />
+    <Exec Command="mdoc --debug assemble -o &quot;$(_MonoDocOutputPath)MonoAndroid-lib&quot; @(_MonoDocAssembly->'&quot;%(SourceFolder)&quot;', ' ')" />
+  </Target>
+  <Target Name="_CopyMonoAndroidDocsSource"
+      Inputs="$(_AndroidApiDocsPath)MonoAndroid-docs.source"
+      Outputs="$(_MonodocOutputPath)MonoAndroid-docs.source">
+    <Copy
+        SourceFiles="$(_AndroidApiDocsPath)MonoAndroid-docs.source"
+        DestinationFolder="$(_MonoDocOutputPath)" />
+  </Target>
+  <Target
+      Name="AssembleApiDocs"
+      DependsOnTargets="_GenerateMsxDocXmls;_GenerateMsxDocXmlRedirects;_GenerateMonoAndroidLibArchive;_CopyMonoAndroidDocsSource"
```

Comment:
One option I considered was to move the `_GenerateMsxDocXmls` and `_GenerateMsxDocXmlRedirects` targets into the `create-vsix.targets` file and the `_GenerateMonoAndroidLibArchive` and `_CopyMonoAndroidDocsSource` targets into the `create-pkg.targets` file. That could make it easy to see which targets are used for which installer. On the other hand, the current approach has the advantage that both invocations of `mdoc` are right next to each other. I'd be happy to split them up into the individual installer targets files if that seems better though.